diesel/src/main/java/com/nuvoex/diesel/core/LoginPresenter.java

```diff
@@ -80,6 +91,35 @@ public int getUserNameLength() {
         return USERNAME_LENGTH;
     }
 
+    @Override
+    public void redirectToPlayStore() {
+        String playStoreID = mLoginView.getApplicationContext().getPackageName();
```

Comment:
Presenter should not contain any Android SDK related code. I think we should move this to the view (Fragment/Activity)